gridstatus/ercot.py

```diff
@@ -453,40 +477,75 @@ def _get_todays_outlook_non_forecast(self, date, verbose=False):
 
         return data.reset_index(drop=True)
 
-    def get_load_forecast(self, date, verbose=False):
-        """Returns load forecast
+    @support_date_range("HOUR_START")
+    def get_load_forecast(
+        self,
+        date,
+        forecast_type: ERCOTSevenDayLoadForecastReport,
```

Comment:
to maintain existing behavior, can we make the `BY_FORECAST_ZONE` report the default parameter? 

this will also partiaily fix some of the ercot tests that are failing